### PR TITLE
Make RejectedHandlerException handlers log warnings consistently

### PR DESCRIPTION
**Pull request by nicktindall**
I imagine this will break a few things as it flushes through. Then again it doesn't seem to be used much so maybe it won't be so bad.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -34,4 +34,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.projectKey=OpenHFT_Chronicle-Network
+        run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.projectKey=OpenHFT_Chronicle-Network -Psonar
```
Comment (nicktindall) [inline .github/workflows/build.yml:37]: Get sonar working too

File: src/test/java/net/openhft/chronicle/network/UberHandlerTest.java [hunk 2]
```diff
@@ -289,7 +290,7 @@ public void onRead(@NotNull WireIn inWire, @NotNull WireOut outWire) {
 
         @Override
         public void onInitialize(WireOut outWire) throws RejectedExecutionException {
-            throw new RejectedExecutionException("Rejected in onInitialize");
+            throw new RejectedHandlerException("Rejected in onInitialize");
```
Comment (nicktindall) [inline src/test/java/net/openhft/chronicle/network/UberHandlerTest.java:293]: This was a typo from previous commit

Comment (nicktindall) [reply src/test/java/net/openhft/chronicle/network/UberHandlerTest.java:293]: That I only noticed now because Sonar is working :tada:

Comment (JerryShea) [reply src/test/java/net/openhft/chronicle/network/UberHandlerTest.java:293]: hehe nice

**Review by JerryShea — APPROVED** [review]

**Pull request merged**